modules/parachains/src/lib.rs

```diff
@@ -383,40 +422,16 @@ pub mod pallet {
 		) -> Result<UpdateParachainHeadArtifacts, ()> {
 			// check if head has been already updated at better relay chain block. Without this
 			// check, we may import heads in random order
+			Self::validate_updated_parachain_head(
+				&stored_best_head,
+				updated_at_relay_block_number,
+				updated_head_hash,
+				format!("The head of parachain {:?} can't be updated", parachain),
```

Comment:
Let's avoid any formatting outside of log calls (I wonder if it compiles actually) in the runtime code

Reply:
Yes, doesn't seem to compile. Done.